src/features/user/CompleteSignup/index.tsx

```diff
@@ -255,380 +255,378 @@ export default function CompleteSignup(): ReactElement | null {
-                defaultValue={true}
-                render={({ field: { onChange, value } }) => {
-                  return (
-                    <NewToggleSwitch
-                      checked={value}
-                      onChange={(e) => onChange(e.target.checked)}
-                      inputProps={{ 'aria-label': 'secondary checkbox' }}
-                      id="get-news"
-                    />
-                  );
+              <NewToggleSwitch
+                checked={acceptTerms || false}
+                onChange={(e) => {
+                  handleTermsAndCondition(e.target.checked);
                 }}
+                inputProps={{ 'aria-label': 'secondary checkbox' }}
+                id="terms"
               />
             </div>
-
-            <div>
-              <div className={styles.inlineToggleGroup}>
-                <div className={styles.mainText}>
-                  <label htmlFor={'terms'} style={{ cursor: 'pointer' }}>
-                    {t.rich('termAndCondition', {
-                      termsLink: (chunks) => (
-                        <a
-                          className="planet-links"
-                          rel="noopener noreferrer"
-                          href={`https://pp.eco/legal/${locale}/terms`}
-                          target={'_blank'}
-                        >
-                          {chunks}
-                        </a>
-                      ),
-                    })}
-                  </label>
-                </div>
-                <NewToggleSwitch
-                  checked={acceptTerms || false}
-                  onChange={(e) => {
-                    handleTermsAndCondition(e.target.checked);
-                  }}
-                  inputProps={{ 'aria-label': 'secondary checkbox' }}
-                  id="terms"
-                />
+            {!acceptTerms && typeof acceptTerms !== 'object' && (
+              <div className={styles.termsError}>
+                {t('termAndConditionError')}
               </div>
-              {!acceptTerms && typeof acceptTerms !== 'object' && (
-                <div className={styles.termsError}>
-                  {t('termAndConditionError')}
-                </div>
-              )}
-            </div>
-            <div className={styles.horizontalLine} />
-
-            <button
-              id={'signupCreate'}
-              className={styles.saveButton}
-              onClick={handleSubmit(createButtonClicked)}
-              disabled={isProcessing}
-            >
-              {submit ? (
-                <div className={styles.spinner}></div>
-              ) : (
-                t('createAccount')
-              )}
-            </button>
+            )}
           </div>
+          <div className={styles.horizontalLine} />
+
+          <button
+            id={'signupCreate'}
+            className={styles.saveButton}
+            onClick={handleSubmit(createButtonClicked)}
+            disabled={isProcessing}
```

Comment:
Show loading state in the button when it is disabled (isProcessing = true).

Reply:
This has been handled in PR #2737.